Automattic/wp-calypso
# Home: Podcast Launched Card

#### Changes proposed in this Pull Request

These changes add a primary card specific to the onboarding experience of a user who has just created a podcast website.

Podcasts | Non-podcasts
--- | ---
<img width="965" alt="Screen Shot 2020-11-30 at 6 03 52 PM" src="https://user-images.githubusercontent.com/66652282/100676383-c584d700-3336-11eb-9efa-9697e809c596.png"> | <img width="969" alt="Screen Shot 2020-11-27 at 17 21 19" src="https://user-images.githubusercontent.com/1233880/100469093-f0bcbb80-30d5-11eb-9c91-b4080a6df025.png">

#### Testing instructions

- Use Calypso Live or checkout this branch.
- Go to My Home for a newly created website.
- Convert the site to a podcasting site:
  - Open a WP shell in your WP.com sandbox: `wp shell --user=<YOUR_WPCOM_USERNAME> --url=<YOUR_SITE_ADDRESS>`.
  - Update the blog option: `update_blog_option( <SITE_ID>, 'anchor_podcast', '3e51070' );`.
- Sandbox the API and reload My Home (this is to prevent our caching mechanism from returning the previous blog option value).
- Ensure that the title of the first site setup task reads as "Welcome to your podcast site!".
- Access the Tracks Live View in MC (PCYsg-5hv-p2).
- Filter the events so only site setup list events for your user are shown (you need to wait at least 5 minutes before they appear):
<img width="530" alt="Screen Shot 2020-11-30 at 13 10 22" src="https://user-images.githubusercontent.com/1233880/100608895-6c547d80-330d-11eb-92ce-9eb7d23f1acb.png">

- Expand the props and make sure there is a new `is_podcasting_site` event prop:
<img width="387" alt="Screen Shot 2020-11-30 at 13 18 16" src="https://user-images.githubusercontent.com/1233880/100609611-9490ac00-330e-11eb-9085-092c3450f7cc.png">

- Clean up the `anchor_podcast` option value from your site:
  - Open a WP shell in your WP.com sandbox: `wp shell --user=<YOUR_WPCOM_USERNAME> --url=<YOUR_SITE_ADDRESS>`.
  - Remove the blog option: `delete_blog_option( <SITE_ID>, 'anchor_podcast' );`.

Fixes 322-gh-dotcom-manage
